test/Sentry.Unity.Editor.Tests/DebugSymbolUploadTests.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+using Sentry.Unity.Editor.Android;
+
+namespace Sentry.Unity.Editor.Tests
+{
+    public class DebugSymbolUploadTests
+    {
+        private class Fixture
+        {
+            public string FakeProjectPath { get; set; }
+            public string UnityProjectPath { get; set; }
+            public string GradleProjectPath { get; set; }
+            public string SentryCliPath { get; set; }
+
+            public Fixture()
+            {
+                FakeProjectPath = Path.GetRandomFileName();
```

Comment:
This will give you a FQPN under the temp directory of the device

```suggestion
                FakeProjectPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString())
```